ndn-svs/svspubsub.cpp

```diff
@@ -284,19 +302,41 @@ SVSPubSub::processMapping(const NodeID& nodeId, SeqNo seqNo)
   bool queued = false;
   for (const auto& sub : m_prefixSubscriptions)
   {
-    if (sub.prefix.isPrefixOf(mapping.first))
+    if (sub.prefix.isPrefixOf(mapping.first) and sub.autofetch)
     {
       m_fetchMap[std::pair(nodeId, seqNo)].push_back(sub);
       queued = true;
     }
+    else if (sub.prefix.isPrefixOf(mapping.first) and !sub.autofetch)
+    {
+      SubscriptionData subData = {
+          mapping.first,
+          ndn::span<const uint8_t>{},
+          nodeId,
+          seqNo,
+          ndn::Data()
```

Comment:
Do you have any suggestion? Thanks!

Reply:
Not at the moment... I didn't do too much thinking to be honest. But *maybe* this should be a new/separate API instead of shoehorning it into the existing API with a bool as discriminator.

Reply:
Yeah... But it's better to be compatible with old APIs because of existing apps...